src/components/MTableEditRow/m-table-edit-row.js

```diff
@@ -171,7 +171,7 @@ export default class MTableEditRow extends React.Component {
 
   renderActions() {
     if (this.props.mode === 'bulk') {
-      return <TableCell padding="none" key="key-actions-column" />;
```

Comment:
Why remove this? Do we not want to return an empty table cell here? I could be wrong - just curious.

@peacechen @Domino987

Reply:
TableCell takes up space even if its contents are empty. For example when the details icon isn't rendered, the icon is gone, but the TableCell consumes a column's worth of space and incorrectly offsets the header.

Reply:
Yes this looks good to me.

Reply:
I understand when we have icons it's incorrectly offsets the header, but with this modification it's render the bulkEdit verry weird :
![Alt Text](https://i.gyazo.com/2c1e334a03cd9961c01a07e7fc9f3c1c.gif) 
when TableCell is removed the th of Action isn't removed,
 idk if i forgot a thing with you'r new option.
Thx

Reply:
Look like there's a bug for the other actions such as Delete not being accounted for in edit mode. I'll investigate and submit a PR. In the meantime, please paste your comment and screenshot into a new ticket so that it can be referenced by the PR.